plugins/notifications-backend/config.d.ts

```diff
@@ -34,6 +34,13 @@ export interface Config {
     defaultSettings?: {
       channels?: {
         id: string;
+        /**
+         * Optional flag to enable/disable the channel by default.
```

Comment:
Please either add comments for all config values or remove this. Alone it really doesn't explain the functionality